Library/ViewModels/PledgeDescriptionViewModel.swift

```diff
@@ -32,10 +33,15 @@ public final class PledgeDescriptionViewModel: PledgeDescriptionViewModelType,
       .skipNil()
       .map { Format.date(secondsInUTC: $0, template: DateFormatter.monthYear, timeZone: UTCTimeZone) }
 
+    self.estimatedDeliveryStackViewIsHidden = self.configDataProperty.signal
+      .skipNil()
+      .map(second)
+      .map { $0.estimatedDeliveryOn.isNil }
+
     self.presentTrustAndSafety = self.learnMoreTappedProperty.signal
-    self.configureRewardCardViewWithData = self.configDataProperty.signal
+    self.rewardTitle = self.configDataProperty.signal
       .skipNil()
-      .map { project, reward in (project, .init(left: reward)) }
+      .map { _, reward in reward.title ?? Strings.Back_it_because_you_believe_in_it() }
```

Comment:
Does it make sense to check if title is `nil` and also whether its `noReward`? Just to be sure 🤔

Reply:
The `NoReward` only contains some of the properties of the `Reward` object, and it doesn't have the `title`. So the title, in this case, will always be `nil`.